docs/draft-ietf-dnsop-dns-catalog-zones.html

```diff
@@ -33,7 +33,7 @@
   this can be consolidated so that style settings occur only in one place, but
   for now the contents of this file consists first of the initial CSS work as
   provided to the RFC Formatter (xml2rfc) work, followed by itemized and
-  commented changes found necssary during the development of the v3
+  commented changes found necessary during the development of the v3
```

Comment:
I imagine this content comes from somewhere. I'd be happy to send a PR to that somewhere if it were to be identified.

Reply:
https://github.com/ietf-tools/xml2rfc/blob/main/xml2rfc/data/xml2rfc.css#L9